### DISCUSSION TITLE
E/E_Common.hpp: No such file or directory

### DISCUSSION THREAD

**Discussion by stdeguzman** [ANLAB-KAIST/KENSv3#46] [Project Q&A 1. Application Layer (Socket)]
I'm trying to debug my code, and it's giving me this error from EchoAssignment.hpp

May I ask for help on how to fix this?

**Comment by Pusnow** [comment]
Your source seems changed. Please, re-download repository.

**Reply by stdeguzman** [reply]
I checked my source and it seems untouched and unchanged. The Common.hpp file is also where it's supposed to be. I still don't understand where the problem is coming from.

**Reply by Pusnow** [reply]
Very strange. Which environment you are using? (cmake version, distro, compiler version, etc.)
